frontend/src/app/forms/steps/pictures-entry/pictures-entry.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input } from '@angular/core';
+import { /*DomSanitizer,*/ SafeResourceUrl } from '@angular/platform-browser';
+import { CameraResultType, CameraSource, Plugins } from '@capacitor/core';
+
+import { FormGroup } from '../../../common/forms/forms';
+import { FormsData } from '../../../model/forms-data.model';
+
+@Component({
+  selector: 'form-pictures-entry',
+  templateUrl: './pictures-entry.component.html',
+  styleUrls: ['./pictures-entry.component.scss'],
+})
+
+/*
+  To save Files as Data.url just change restultType and uncomment everything in this file
+*/
+export class PicturesEntryComponent {
+  constructor(/*private sanitizer: DomSanitizer*/) {}
+  @Input()
+  formsData: FormGroup<FormsData>;
+
+  photo: SafeResourceUrl;
+
+  async takePicture(): Promise<void> {
+    const image = await Plugins.Camera.getPhoto({
+      quality: 100,
+      allowEditing: false,
+      resultType: CameraResultType.Base64,
```

Comment:
I have a doubt. In networks, I could able to see blob object being created even though resultType is set as Base 64. I think it is generated by the capacitor for image preview. Is it the general behaviour? Irrespective of the type blob will be created? If  so will it also be revoked automatically? I hope it should be just because it was created by capacitor. 

Also will there be any configuration we can have for image size? Will always same size  image be returned? Or do we need to have any max width in CSS to restrict the size.

Reply:
I will do some research about this.

Reply:
Added the max-width, so picture gets always resized. ( this will be uploaded tomorrow)